styles.css

```diff
@@ -389,7 +455,7 @@ nav ul li {
 }
 
 .contact-form {
-  margin-top: 0;
+  margin-top: 240px;
   width: 375px;
   height: 718px;

```

Comment:
Kindly update the styling of your contact section to match the figma template better🙏

Your design:
<img width="202" alt="image" src="https://user-images.githubusercontent.com/37116322/221988242-c33f9484-aa37-4a08-a6fe-1bb69f985586.png">


Figma design:
<img width="173" alt="image" src="https://user-images.githubusercontent.com/37116322/221989207-8ac5aa28-caee-4b8b-9102-dddc9077e6e4.png">

A few things to note:
- Please resize your background image so that the shapes are in identical positions to the Figma design
- Please add the border-radius at the top left of the contact section
- Please make sure the paragraph is center aligned
- Kindly update your get in touch button to include interaction styling when the button is hovered and pressed. You can use the `:hover` and `:active` pseudo-selectors. Please make sure to follow the style guide by Figma
<img width="278" alt="image" src="https://user-images.githubusercontent.com/37116322/221989006-ccd27ca0-83ae-49ca-876c-ba0c39aaf67a.png">